code/scripting/lua/LuaReference.cpp

```diff
@@ -80,10 +80,13 @@ bool UniqueLuaReference::isValid() const {
 	return true;
 }
 
-void UniqueLuaReference::pushValue(lua_State* thread) const
+void UniqueLuaReference::pushValue(lua_State* thread, bool guaranteeStack) const
 {
 	Assertion(thread != nullptr, "Valid thread state must be specified!");
 
+	if(!guaranteeStack)
+		lua_checkstack(thread, 1);
```

Comment:
In this case, I would say that curly braces are needed since they are used just a few lines below this.